layers/+lang/python/packages.el

```diff
@@ -238,6 +238,7 @@
       (defun python-default ()
         (setq mode-name "Python"
               tab-width 4
+              comment-inline-offset 2
               fill-column python-fill-column
```

Comment:
Yeah, you're right, spent one hour diving into the details of `(make-local-variable)`, (buffer-local-variables)`, `(setq-default)`, `(setq-local)`, etc. Repushed code again.